Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM arm64v8/node:17.3-bullseye
+EXPOSE tcp/28070
```

Comment:
[NOT BLOCKING]

From what I've seen, it looks like portnumber/tcp is the common practice
It works, but I was wondering if there was any reason for tcp/28070? 

https://docs.docker.com/config/containers/container-networking/
https://www.ctl.io/developers/blog/post/docker-networking-rules